src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -56,40 +56,41 @@ public void resetData(ReadOnlyAddressBook newData) {
         setPersons(newData.getPersonList());
     }
 
-    //// person-level operations
+    //// question-level operations
 
     /**
-     * Returns true if a person with the same identity as {@code person} exists in the address book.
+     * Returns true if a question with the same identity as {@code question} exists in the address book.
      */
-    public boolean hasPerson(Person person) {
-        requireNonNull(person);
-        return persons.contains(person);
+    public boolean hasPerson(Question question) {
+        requireNonNull(question);
+        return persons.contains(question);
     }
 
     /**
-     * Adds a person to the address book.
-     * The person must not already exist in the address book.
+     * Adds a question to the address book.
+     * The question must not already exist in the address book.
      */
-    public void addPerson(Person p) {
+    public void addPerson(Question p) {
         persons.add(p);
     }
 
     /**
-     * Replaces the given person {@code target} in the list with {@code editedPerson}.
+     * Replaces the given question {@code target} in the list with {@code editedQuestion}.
      * {@code target} must exist in the address book.
-     * The person identity of {@code editedPerson} must not be the same as another existing person in the address book.
+     * The question identity of {@code editedQuestion} must not be the same as another existing question
+     * in the address book.
      */
-    public void setPerson(Person target, Person editedPerson) {
-        requireNonNull(editedPerson);
+    public void setPerson(Question target, Question editedQuestion) {
+        requireNonNull(editedQuestion);
 
-        persons.setPerson(target, editedPerson);
+        persons.setPerson(target, editedQuestion);
     }
 
     /**
      * Removes {@code key} from this {@code AddressBook}.
      * {@code key} must exist in the address book.
      */
-    public void removePerson(Person key) {
+    public void removePerson(Question key) {
```

Comment:
same here. maybe change from person to Question?

Reply:
edited